include/zephyr/bluetooth/audio/audio.h

```diff
@@ -93,6 +93,77 @@ enum bt_audio_parental_rating {
 	BT_AUDIO_PARENTAL_RATING_AGE_18_OR_ABOVE  = 0x0F
 };
 
+/**
+ * @brief Response Status Code
+ *
+ * These are sent by the server to the client when a stream operation is
+ * requested.
+ */
+enum bt_ascs_rsp_code {
+	/** Server completed operation successfully */
+	BT_ASCS_RSP_CODE_SUCCESS              = 0x00,
+	/** Server did not support operation by client */
+	BT_ASCS_RSP_CODE_NOT_SUPPORTED        = 0x01,
+	/** Server rejected due to invalid operation length */
+	BT_ASCS_RSP_CODE_INVALID_LENGTH       = 0x02,
```

Comment:
I am not convinced about the naming. I know that those values come from ASCS specification but in audio.h API we don't use the ASCS prefix

Reply:
Me neither. 

That's also a challenge I had with BASS; each service may define a set of error/response codes, and we can't just prefix them with `bt_bap` or `bt_audio`, as they differ. 

We could of course convert them to e.g. errno values or something as well. 

So basically should we expose service specific error codes in the public API at all? And if so, how should we group/prefix them?

Reply:
> So basically should we expose service specific error codes in the public API at all? And if so, how should we group/prefix them?

We could also have `BT_BAP_ERR` codes, and transform from BASS and ASCS to a public BT_BAP one.